Makefile

```diff
@@ -76,6 +79,12 @@ SIMPLEDOCS = docs/faq.inc docs/fips.inc \
 	     docs/manpages.html \
 	     docs/mansidebar.html
 
+GLOSSARY=$(CHECKOUTS)/general-policies/policies/glossary.md
+all_GENERAL_POLICIES=$(wildcard $(CHECKOUTS)/general-policies/policies/*.md)
+all_TECHNICAL_POLICIES=$(wildcard $(CHECKOUTS)/technical-policies/policies/*.md)
+GENERAL_POLICIES=$(filter-out $(CHECKOUTS)/general-policies/policies/README.md $(GLOSSARY),$(all_GENERAL_POLICIES))
+TECHNICAL_POLICIES=$(filter-out $(CHECKOUTS)/general-policies/policies/README.md,$(all_TECHNICAL_POLICIES))
```

Comment:
That should be fixed.

Reply:
Fixed by changing `general-policies` to `technical-policies`.  Other than that, I'd prefer to keep this, just in case someone adds a README.md at some point